.gitignore

```diff
@@ -1,3 +1,4 @@
-vendor
-composer.lock
+.idea/
```

Comment:
this should be removed. It is system-specific, not project-specific. Use a rule local to your machine (potentially system-wide to configure it only once for all projects you contribute too): https://help.github.com/articles/ignoring-files/